.gitignore

```diff
@@ -171,3 +171,6 @@ thumbs.db
 node_modules/
 
 /.luarc.json
+
+#in-house-ctf
+examples/agents/in-house-ctf/scratch
```

Comment:
Let's move the `.gitignore` into the same directory as the data (which will be `benchmarks/in-house-ctf`, see comment below)